@xen-orchestra/backups/writers/_ContinuousReplicationWriter.js

```diff
@@ -2,17 +2,20 @@ const { asyncMap, asyncMapSettled } = require('@xen-orchestra/async-map')
 const ignoreErrors = require('promise-toolbox/ignoreErrors')
 const { formatDateTime } = require('@xen-orchestra/xapi')
 
-const { formatFilenameDate } = require('./_filenameDate')
-const { getOldEntries } = require('./_getOldEntries')
-const { importDeltaVm, TAG_COPY_SRC } = require('./_deltaVm')
+const { formatFilenameDate } = require('../_filenameDate')
+const { getOldEntries } = require('../_getOldEntries')
+const { importDeltaVm, TAG_COPY_SRC } = require('../_deltaVm')
+const { Task } = require('../Task')
+
+const { AbstractDeltaWriter } = require('./_AbstractDeltaWriter')
+const { AbstractReplicationWriter } = require('./_AbstractReplicationWriter')
 const { listReplicatedVms } = require('./_listReplicatedVms')
-const { Task } = require('./Task')
 
-exports.ContinuousReplicationWriter = class ContinuousReplicationWriter {
+exports.ContinuousReplicationWriter = class ContinuousReplicationWriter extends (
```

Comment:
What do  you think of renaming this to `DeltaReplicationWriter` and `DisasterRecoveryWriter` to `FullReplicationWriter`?

Reply:
Yes it's better :+1: